CoroutineTemplate/build.gradle

```diff
@@ -18,13 +18,19 @@ buildscript {
         constraint_layout_version = '2.0.0-alpha3'
 
         hilt_version = '2.33-beta'
-        
+
         kotlin_version = '1.4.30'
         kotlinx_coroutines_version = '1.4.1'
 
+        navigation_version = '2.2.0-rc04'
```

Comment:
Could we upgrade the version to `2.3.4` ? 🙏 

https://developer.android.com/jetpack/androidx/releases/navigation

Reply:
No biggie, but shall we rename it to `androidx_navigation_version` ?